src/config/jest.config.js

```diff
@@ -59,7 +54,7 @@ const setupFiles = [
 ]
 for (const setupFile of setupFiles) {
   if (hasFile(setupFile)) {
-    jestConfig.setupFilesAfterEnv = [fromRoot(setupFile)]
+    jestConfig.setupFilesAfterEnv = `<rootDir>/${setupFile}`
```

Comment:
https://github.com/testing-library/dom-testing-library/runs/3611964206
```
[test] ● Validation Error:
[test] 
[test]   Option "setupFilesAfterEnv" must be of type:
[test]     array
[test]   but instead received:
[test]     string
[test] 
[test]   Example:
[test]   {
[test]     "setupFilesAfterEnv": [
[test]       "<rootDir>/testSetupFile.js"
[test]     ]
[test]   }
[test] 
[test]   Configuration Documentation:
[test]   https://jestjs.io/docs/configuration
```

Reply:
Fixed in #221

Reply:
Oops I forgot to test that use case, thanks for the fix!